helm_deploy/hmpps-assess-risks-and-needs/values.yaml

```diff
@@ -58,131 +57,125 @@ generic-service:
-    pingdom-30: "52.67.148.55"
-    pingdom-31: "52.73.209.122"
-    pingdom-32: "52.89.43.70"
-    pingdom-33: "52.194.115.181"
-    pingdom-34: "52.197.31.124"
-    pingdom-35: "52.197.224.235"
-    pingdom-36: "52.198.25.184"
-    pingdom-37: "52.201.3.199"
-    pingdom-38: "52.209.34.226"
-    pingdom-39: "52.209.186.226"
-    pingdom-40: "52.210.232.124"
-    pingdom-41: "54.68.48.199"
-    pingdom-42: "54.70.202.58"
-    pingdom-43: "54.94.206.111"
-    pingdom-44: "64.237.49.203"
-    pingdom-45: "64.237.55.3"
-    pingdom-46: "66.165.229.130"
-    pingdom-47: "66.165.233.234"
-    pingdom-48: "72.46.130.18"
-    pingdom-49: "72.46.131.10"
-    pingdom-50: "76.72.167.154"
-    pingdom-51: "76.72.172.208"
-    pingdom-52: "76.164.234.106"
-    pingdom-53: "76.164.234.130"
-    pingdom-54: "82.103.136.16"
-    pingdom-55: "82.103.139.165"
-    pingdom-56: "82.103.145.126"
-    pingdom-57: "85.195.116.134"
-    pingdom-58: "89.163.146.247"
-    pingdom-59: "89.163.242.206"
-    pingdom-60: "94.75.211.73"
-    pingdom-61: "94.75.211.74"
-    pingdom-62: "94.247.174.83"
-    pingdom-63: "96.47.225.18"
-    pingdom-64: "103.10.197.10"
-    pingdom-65: "103.47.211.210"
-    pingdom-66: "104.129.24.154"
-    pingdom-67: "104.129.30.18"
-    pingdom-68: "107.182.234.77"
-    pingdom-69: "148.72.170.233"
-    pingdom-70: "148.72.171.17"
-    pingdom-71: "151.106.52.134"
-    pingdom-72: "159.122.168.9"
-    pingdom-73: "162.208.48.94"
-    pingdom-74: "162.218.67.34"
-    pingdom-75: "162.253.128.178"
-    pingdom-76: "168.1.203.46"
-    pingdom-77: "169.51.2.18"
-    pingdom-78: "169.54.70.214"
-    pingdom-79: "169.56.174.151"
-    pingdom-80: "172.241.112.86"
-    pingdom-81: "173.248.147.18"
-    pingdom-82: "173.254.206.242"
-    pingdom-83: "174.34.156.130"
-    pingdom-84: "175.45.132.20"
-    pingdom-85: "178.162.206.244"
-    pingdom-86: "178.255.152.2"
-    pingdom-87: "178.255.153.2"
-    pingdom-88: "179.50.12.212"
-    pingdom-89: "184.75.208.210"
-    pingdom-90: "184.75.209.18"
-    pingdom-91: "184.75.210.90"
-    pingdom-92: "184.75.210.226"
-    pingdom-93: "184.75.214.66"
-    pingdom-94: "184.75.214.98"
-    pingdom-95: "185.39.146.214"
-    pingdom-96: "185.39.146.215"
-    pingdom-97: "185.70.76.23"
-    pingdom-98: "185.93.3.65"
-    pingdom-99: "185.136.156.82"
-    pingdom-100: "185.152.65.167"
-    pingdom-101: "185.180.12.65"
-    pingdom-102: "185.246.208.82"
-    pingdom-103: "188.172.252.34"
-    pingdom-104: "190.120.230.7"
-    pingdom-105: "196.240.207.18"
-    pingdom-106: "196.244.191.18"
-    pingdom-107: "196.245.151.42"
-    pingdom-108: "199.87.228.66"
-    pingdom-109: "200.58.101.248"
-    pingdom-110: "201.33.21.5"
-    pingdom-111: "207.244.80.239"
-    pingdom-112: "209.58.139.193"
-    pingdom-113: "209.58.139.194"
-    pingdom-114: "209.95.50.14"
-    pingdom-115: "212.78.83.12"
-    pingdom-116: "212.78.83.16"
+    hmpps-daresbury-park: 194.168.183.128/28
+    pingdom-1: 5.172.196.188/32
```

Comment:
not sure you need pingdom ips, most services route pingdom via the [healthkick app](https://github.com/ministryofjustice/health-kick)